README.md

```diff
@@ -42,6 +42,28 @@ For example:
 When filing bug reports with unit test output, always use Maven
 option `-B` to avoid control characters in output.
 
+## Docker image
+
+At Docker Hub:
+
+* https://hub.docker.com/r/folioorg/okapi released versions
+* https://hub.docker.com/r/folioci/okapi snapshot versions
+
+See [Automation/Docker
+Hub](https://dev.folio.org/guides/automation/#docker-hub) for details.
+
+Docker images are the primary distribution model for FOLIO modules.
+To run the images you will need the Docker Engine or Docker Desktop
+runtime.
+
+## Ubuntu package
+
+Import the FOLIO signing key and add the [FOLIO apt
```

Comment:
This is not for any Ubuntu.. Just Ubuntu focal 20.04 LTS.. So perhaps header should be changed or there could be a leading paragraph saying that.

Reply:
If prefer not to mention the Ubuntu release in the header so that links to this section will not break.

Clicking on https://repository.folio.org/packages/ubuntu/ shows that we have packages for xenial and focal. However, latest packages are available for focal only.

I've added "Ubuntu 20.04 LTS (Focal Fossa)" to the text.